### PR TITLE
added new line in demo.txt

### PR DESCRIPTION
**Pull request by amitaryan0010**

### CODE REVIEW

File: demo.txt [hunk 1]
```diff
@@ -1 +1,4 @@
-Initial Content
+Initial Content changed
```
Comment (cloudtechmasters) [inline demo.txt:1]: why is this change?

File: demo.txt [hunk 2]
```diff
@@ -1 +1,4 @@
...
+
+
+Added new line
```
Comment (cloudtechmasters) [inline demo.txt:4]: why this new one is needed?

File: demo.txt (hunk 1 above)
Comment (amitaryan0010) [reply demo.txt:1]: Added new line for our code.

File: demo.txt (hunk 2 above)
Comment (amitaryan0010) [reply demo.txt:4]: end of code

**Review by cloudtechmasters — APPROVED** [review]

**Pull request merged**